### PR TITLE
Fix/layout styles

### PR DESCRIPTION
**Pull request by partialdigital**

### CODE REVIEW

File: packages/app/src/components/header/LinksPopover.vue [hunk 1]
```diff
@@ -3,7 +3,8 @@
     <MenuButton>
       <div class="navigation-link" :class="{ active: open }">
         {{ label }}
-        <ChevronDownIcon class="dropdown-icon" aria-hidden="true" />
+        <!-- <ChevronDownIcon class="dropdown-icon" aria-hidden="true" /> -->
```
Comment (mc-ducky) [inline packages/app/src/components/header/LinksPopover.vue:6]: let's completely remove

File: packages/app/src/components/header/TheHeader.vue [hunk 2]
```diff
@@ -4,13 +4,14 @@
...
-            <span class="text-[46px]">
+            <img src="/images/logo-sophon-testnet.svg" />
+            <!-- <span class="text-[46px]">
               <sophon />
             </span>
             <span class="text-2xl font-medium">Sophon</span>
             <span class="top-[2px] rounded-full bg-blue px-2 text-2xs/4 font-semibold uppercase text-white"
               >Testnet</span
-            >
+            > -->
```
Comment (mc-ducky) [inline packages/app/src/components/header/TheHeader.vue:14]: same here, let's completely remove

File: packages/app/src/components/header/LinksPopover.vue (hunk 1 above)
Comment (partialdigital) [reply packages/app/src/components/header/LinksPopover.vue]: yes yes, all done

File: packages/app/src/components/header/TheHeader.vue (hunk 2 above)
Comment (partialdigital) [reply packages/app/src/components/header/TheHeader.vue]: samesame

**Pull request merged**